webapp/channels/src/components/post_deleted_modal.tsx

```diff
@@ -46,18 +48,13 @@ const PostDeletedModal = ({
                 </p>
             </Modal.Body>
             <Modal.Footer>
-                <button
-                    type='button'
-                    className='btn btn-primary'
-                    autoFocus={true}
```

Comment:
@hmhealey Do you think this autofocus is really needed? I think it is only used here, and I would prefer to keep the interface "cleaner".

Reply:
It might keep the interface cleaner, but removing it would impact UX, so I don't think we should remove it. There's some bigger things where I think we may need to make the UX conform to how the existing component behaves, but I don't think it's worth fighting with UX on this one.

That said, we should also review how we use `autoFocus` since it can cause accessibility problems by hijacking the focus unexpectedly, so we may end up removing it later for that reason